bsc-plugin/src/lib/rooibos/FileFactory.ts

```diff
@@ -51,13 +51,14 @@ export class FileFactory {
             let fileContents = fs.readFileSync(sourcePath, 'utf8');
             let destPath = path.join(this.targetPath, `${fileName}.bs`);
             let entry = { src: sourcePath, dest: destPath };
-
+            //BRON_AST_EDIT_HERE
             this.addedFrameworkFiles.push(program.setFile(entry, fileContents));
```

Comment:
@georgejecook these framework files are static, right? As in, they don't change as part of every watch cycle? Rather than adding (and re-parsing) them every time in beforeProgramTranspile, and then removing them every afterProgramTranspile, I'm thinking it's ok to leave this code as-is. That'll save time not re-parsing them every time.

Reply:
yes. correct.